### PR TITLE
RELATED: SD-884 upgrade typings

### PR DESCRIPTION
**Pull request by lanhq**
---

**Proposed release type:** patch

# PR checklist

- [x] Change was tested using dev-release in Analytical Designer and Dashboards (if applicable).
- [x] Change is described in [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] Migration guide (for major update) is written to [CHANGELOG.md](../blob/master/CHANGELOG.md).
- [x] The proposed release type is appropriate (see the comment in [PR template](../blob/master/.github/pull_request_template.md))

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/97/ : SUCCESS in 3s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/533/ : SUCCESS in 1m 52s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-oss-check/98/ : SUCCESS in 4s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/534/ : SUCCESS in 43s

File: CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,11 @@ The REST API versions in the table are just for your information as the values a
 |\>= 10.0.0|3
 |<= 9.0.1|2
 
+<a name="12.5.4"></a>
+## 2020-04-20 Version [12.5.4](https://github.com/gooddata/gooddata-js/compare/v12.5.3...v12.5.4)
```
Comment (ptquang86) [inline CHANGELOG.md:17]: where is version `12.5.3`?

Please take a look at the order of versions in the previous link, 
it is from `v12.5.2` => `v12.5.1` (newer to older version)
`https://github.com/gooddata/gooddata-js/compare/v12.5.2...v12.5.1`

Comment (lanhq) [reply CHANGELOG.md:17]: - previous link is incorrect - please check the second previous link
- 12.5.3 is latest version was released which isn't mentioned in CHANGELOG.md

**Comment by naibinh** [comment]
LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (naibinh) [reply CHANGELOG.md:17]: Yes, v12.5.3 is [here](https://www.npmjs.com/package/@gooddata/gooddata-js/v/12.5.3) on npm registry

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-js-merge-compliance-check/123/ : SUCCESS in 22s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-js-unit-tests-zuul-docker-el7/536/ : SUCCESS in 1m 39s

**Pull request merged**